docs/self-managed/about-self-managed.md

```diff
@@ -7,29 +7,38 @@ title: "Camunda 8 Self-Managed"
-
 :::
 
-As an alternative to using Camunda 8 through SaaS, you can host it yourself; we call this setup Camunda 8 Self-Managed.
+Camunda 8 Self-Managed is a self-hosted alternative to using Camunda 8 SaaS. Building process automation solutions with Camunda 8 works similarly regardless of hosting and deployment.
 
-Building process automation solutions with Camunda 8 Self-Managed is similar to working with Camunda 8 SaaS. For more information on Camunda 8 SaaS, visit [What is Camunda 8?](../components/concepts/what-is-camunda-8.md) If you are new to Camunda 8, we recommend you start your journey with [Camunda 8 SaaS-based guides](../../guides/).
+For more information on Camunda 8 SaaS, visit [What is Camunda 8?](../components/concepts/what-is-camunda-8.md) If you are new to Camunda 8, we recommend you start your journey with [Camunda 8 SaaS-based guides](../../guides/).
 
 The content in this section of the documentation includes:
 
 - Everything you need to download, configure, and work with each component of Camunda 8 Self-Managed.
 - Features specific to Camunda 8 Self-Managed.
 
-The following components are available for Camunda 8 Self-Managed:
+To get started with your Self-Managed installation, see our [installation overview](./setup/overview.md).
+
+## Architecture
+
+Camunda 8 Self-Managed consists of multiple web applications and gRPC services. The following example architecture diagram uses two [Ingress objects](./setup/guides/ingress-setup.md#combined-ingress-setup) to access these services externally: an Ingress with HTTP(S) protocol for all web applications using a single domain, and another Ingress with gRPC protocol for the Zeebe workflow engine.
+
+![Camunda 8 Self-Managed Architecture Diagram - Combined Ingress](./assets/camunda-platform-8-self-managed-architecture-diagram-combined-ingress.png)
+
+:::note
+By default, the deployment requires at least three Kubernetes Nodes to meet the Pod Anti-Affinity constraints.
+:::
+
+In this configuration, Camunda 8 Self-Managed can be accessed as follows:
+
+- Web applications: `https://camunda.example.com/[identity|operate|optimize|tasklist|modeler]`
+  - Web Modeler also exposes a WebSocket endpoint on `https://camunda.example.com/modeler-ws`. This is only used by the application itself and not supposed to be accessed by users directly.
+- Keycloak authentication: `https://camunda.example.com/auth`
+- Zeebe gateway: `grpc://zeebe.camunda.example.com`
+
+## Availability
 
-- Zeebe
-- Zeebe Gateway
-- Operate
-- Tasklist
-- Connectors
-- Optimize
-- Identity (not available in Camunda 8 SaaS)
-- Console [<span class="badge badge--enterprise-only">Enterprise only</span>](../../reference/licenses/#console)
-- Web Modeler
+Each component of Camunda 8 Self-Managed can be deployed in single or high-availability mode, which provides minimal to no service interruption. The following diagram represents deployment in high-availability mode:
```

Comment:
I would argue that 'it is possible to deploy HA mode', and therefore the statement is correct :P I feel it would sound more confusing if we state its not HA or partially.

Would it be an idea to create a follow up to create a guide or make the HCs configuration more easy for customers to set up all apps in HA mode then?

Reply:
@hisImminence @theburi Do you think this diagram with a theoretical HA mode is adding anything for a technical audience? I could remove it if it's only adding to confusion, and make an issue to create the follow-up HA guide. Or, leave it for now, and make that guide to link to from here when it's done.

Reply:
>  I could remove it if it's only adding to confusion, and make an issue to create the follow-up HA guide

You are referring to the diagram on [this page](https://docs.camunda.io/docs/self-managed/setup/install/), or? 

I would go with this option: I could remove it if it's only adding to confusion, and make an issue to create the follow-up HA guide.

Reply:
Specifically the high availability diagram [here](https://docs.camunda.io/docs/self-managed/platform-architecture/overview/#deployment). I've gone ahead and removed it in this iteration, and set up [this issue](https://github.com/camunda/camunda-docs/issues/4080) to look into it outside of this cleanup PR.